docs/open-api/openapi.yaml

```diff
@@ -475,6 +497,7 @@ components:
           - "COLUMN"
           - "FILESET"
           - "TOPIC"
+          - "MODEL"
```

Comment:
also need add "MODEL" to here:
https://github.com/apache/gravitino/blob/a58b7f8f0e629a9a2b8a26df821e25ee0e6c0493/docs/open-api/owners.yaml#L184-L191

Reply:
Currently, we don't support tag and access-control and tag related operations, so I don't think we need to modify this for now.